www/apps/resources/app/commerce-modules/auth/examples/page.mdx

```diff
@@ -312,7 +312,7 @@ This example uses the [jsonwebtoken NPM package](https://www.npmjs.com/package/j
     const authModuleService = await initializeAuthModule()
 
     return NextResponse.json({
-      auth_users: await authModuleService.list(),
+      auth_identitys: await authModuleService.list(),
```

Comment:
```suggestion
      auth_identities: await authModuleService.list(),
```

Made the same find-and-replace mistake 😄 There are few more occurences like that.